go/kbfs/libkbfs/kbfs_ops_concur_test.go

```diff
@@ -84,7 +85,8 @@ func TestKBFSOpsConcurDoubleMDGet(t *testing.T) {
 	c := make(chan error, n)
 	cl := &CounterLock{}
 	ops := getOps(config, rootNode.GetFolderBranch().Tlf)
-	ops.mdWriterLock.locker = cl
+	ops.mdWriterLock = kbfssync.MakeLeveledMutex(
+		kbfssync.MutexLevel(fboMDWriter), cl)
```

Comment:
Is this a bugfix?

Reply:
Not a bugfix as far as I know, just a cleanup of a hacky solution.